### ISSUE TITLE
Quokka Facts

### ISSUE THREAD

**Issue by amykapernick** [amykapernick/quokkas#5]
I'm going to add Quokka facts to the bot soon and need to collect some more facts to get started!

Add your facts to [`/resources/data/facts.js`](https://github.com/amykapernick/quokkas/tree/prod/resources/data) via PR for Hacktoberfest!

Note we will only be accepting one PR per person so this, to keep Hacktoberfest fair

**Comment by dhruvx24** [comment]
@amykapernick  Can I take this up?

**Comment by amykapernick** [comment]
@parikhdhruv24791 yep this is open for anyone to add facts, this issue will stay open for the month so people can keep adding

**Comment by Keatnuxsuo** [comment]
Can I still add more facts, please?
